proto/initia/mstaking/v1/staking.proto

```
@@ -1,6 +1,7 @@
 syntax = "proto3";
 package initia.mstaking.v1;
 
+import "amino/amino.proto";
```

Buf (1.47.2)

import "amino/amino.proto": file does not exist

(COMPILE)